### PR TITLE
Updated Requirements

### PR DESCRIPTION
**Pull request by ericadohring**
Updated flake lint warnings
/platform @cgil

### CODE REVIEW

File: manage.py [hunk 1]
```diff
@@ -36,5 +38,6 @@ def list_routes():
 def make_shell_context():
     return dict(app=app, db=db)
 
+
```
Comment (cgil) [inline manage.py:41]: Why did you add a new line here?

Comment (ericadohring) [reply manage.py:41]: I needed 2 newlines to pass the tests

**Comment by cgil** [comment]
looks good to me!
